### PR TITLE
Update syntax and dependencies for Elixir 1.9

### PR DESCRIPTION
**Pull request by Stratus3D**
[Card](https://rentpath.atlassian.net/browse/SRV-1159)

The mix.exs file for this library was generating some compiler warnings on Elixir 1.9, so I fixed the syntax and upgraded a few of the dependencies to work with Elixir 1.9.

I'm still testing this with Elixir 1.9, so additional changes may be made.

These changes will probably released as version 1.1.0.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -38,7 +38,7 @@ defmodule ExOps.Mixfile do
     [
       {:ex_doc, ">= 0.0.0", only: :dev},
       {:excoveralls, "~> 0.10", only: :test},
-      {:credo, "~> 0.10", only: [:dev, :test]},
+      {:credo, "~> 1.1.5", only: [:dev, :test]},
```
Comment (Stratus3D) [inline mix.exs]: Older versions of credo generated compiler warnings on Elixir 1.9, so I upgraded to the latest version of credo.

File: mix.exs [hunk 2]
```diff
@@ -4,8 +4,8 @@ defmodule ExOps.Mixfile do
   def project do
     [
       app: :ex_ops,
-      version: "1.0.0",
-      elixir: "~> 1.6",
+      version: "1.1.0",
```
Comment (Stratus3D) [inline mix.exs]: Not sure what our versioning scheme is, but it seems like either the minor or patch versions should be incremented.

File: mix.exs [hunk 3]
```diff
@@ -67,9 +67,9 @@ defmodule ExOps.Mixfile do
   defp preferred_cli_env do
     [
       coveralls: :test,
-      "coveralls.detail": :test,
-      "coveralls.post": :test,
-      "coveralls.html": :test
+      'coveralls.detail': :test,
+      'coveralls.post': :test,
+      'coveralls.html': :test
```
Comment (Stratus3D) [inline mix.exs]: The double quotes are for strings, but keyword lists only allow atom keys. This was what was generating the compiler warning.

Comment (doomspork) [reply mix.exs]: Double quotes followed by a colon is a common way to handle atoms with non-alpha characters in them. In fact, in IEX these single quotes are turned back into double quote:

```elixir
iex> %{'hi.trevor': ":wave:"}
%{"hi.trevor": ":wave:"}

```

File: mix.exs (hunk 1 above)
Comment (doomspork) [reply mix.exs]: You may want to consider making this `~> 1.1` so you won't need to make any changes beyond mix update when the next `1.X` release is out.

Comment (Stratus3D) [reply mix.exs]: Thanks! Good idea.

File: mix.exs (hunk 3 above)
Comment (Stratus3D) [reply mix.exs]: Howdy! Hope things are going well.

I must have mis-read compiler warning 🤔 . After reverting these changes it doesn't complain about anything.

Comment (doomspork) [reply mix.exs]: That's great to hear @Stratus3D! I'm glad you're kicking arse and taking names :grin:

File: mix.exs [hunk 4]
```diff
@@ -38,7 +38,7 @@ defmodule ExOps.Mixfile do
     [
       {:ex_doc, ">= 0.0.0", only: :dev},
       {:excoveralls, "~> 0.10", only: :test},
-      {:credo, "~> 0.10", only: [:dev, :test]},
+      {:credo, "~> 1.1", only: [:dev, :test], runtime: false},
```
Comment (Stratus3D) [inline mix.exs]: Credo should not be a runtime dependency.

File: mix.exs [hunk 5]
```diff
@@ -4,8 +4,8 @@ defmodule ExOps.Mixfile do
   def project do
     [
       app: :ex_ops,
-      version: "1.0.0",
-      elixir: "~> 1.6",
+      version: "1.1.0",
+      elixir: "~> 1.8",
```
Comment (Stratus3D) [inline mix.exs]: Updated this to work with Elixir 1.9, but everything here should work with 1.8 without any issues.

**Pull request merged**